src/components/Globalstats.js

```diff
@@ -1,32 +1,40 @@
-/* eslint-disable */
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
+import { FaArrowRight } from 'react-icons/fa';
+import { Spinner } from 'react-bootstrap';
 import { fetchCovidData } from '../redux/statisticsSlice';
-import '../App.css'
-import { showDetails } from '../redux/statisticsSlice';
+import CountryStats from './CountryStats';
+import '../App.css';
 
 const Globalstats = () => {
-  const { dataArray/*, isLoading */ } = useSelector((state) => state.Covid);
-    const dispatch = useDispatch();
-  
-  const handleClick = (country) => {
-    dispatch(showDetails(country))
-  }  
-    if (dataArray.length === 0) {
-        setTimeout(() => {
-          dispatch(fetchCovidData());
-        }, '1000');
-      }
-    
+//  const [query, setQuery] = useState('');
+  // const Search = (event) => {
+  //   setQuery(event.target.value);
+  // };
+  const { dataArray, isLoading } = useSelector((state) => state.Covid);
+
+  const filteredData = dataArray.filter((data) => (
+    data.country.toLowerCase()
+    /* .includes(query.toLowerCase()) */
+  ));
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchCovidData());
+  }, [dispatch]);
+
+  if (isLoading) {
+    return <Spinner animation="border" />;
+  }
+
   return (
-    <div className='countryContainer'>
-        {dataArray.map((data) => (
-            <div key={data.country+data.region} className="countryCard" onClick={() => handleClick(data.country+data.region)}>
-                <h2>{data.country}</h2>
-                <h3>{data.region}</h3>
-            </div>
-        ))
-}
-    </div>
-  )}
+    <ul className="itemList">
+      {filteredData.map((data) => (
+        <CountryStats key={data.id} details={data} />
+      ))}
```

Comment:
- [ ] Great job working on the Home page but I think it would be great if you add the section between the nav and the data displayed from the API since in the given design there is an image and in the right a text you can anything you want but it is important to make the design like the given design layout 🔥💯🤩🙏🏻

- your
![image](https://github.com/dazehinn/world-covid-19-data/assets/58553711/58e8ca3b-6b7e-4ed4-b175-5a351a5f4be1)

- Given
![image](https://github.com/dazehinn/world-covid-19-data/assets/58553711/edf4422a-2865-45f3-a302-b6ea086fe576)